benchmarks/perception/a3_stereo_image_proc/src/disparity_output_component.cpp

```diff
@@ -0,0 +1,55 @@
+#include <rclcpp/rclcpp.hpp>
+#include <image_transport/image_transport.hpp>
+#include <sensor_msgs/msg/camera_info.hpp>
+#include <sensor_msgs/msg/image.hpp>
+#include <stereo_msgs/msg/disparity_image.hpp>
+
+#include "tracetools_benchmark/tracetools.h"
+#include "a3_stereo_image_proc/disparity_output_component.hpp"
+
+#include "rclcpp_components/register_node_macro.hpp"
+
+namespace robotperf
+{
+
+namespace perception
+{
+
+DisparityOutputComponent::DisparityOutputComponent(const rclcpp::NodeOptions & options)
+: rclcpp::Node("DisparityOutputComponent", options)
+{
+
+  //Create subscriber to Disparity image with callback function
+  sub_disparity_ = this->create_subscription<stereo_msgs::msg::DisparityImage>(
+    "/benchmark/disparity", rclcpp::QoS(10), std::bind(&DisparityOutputComponent::disparityCb, this, std::placeholders::_1));
+
+}
+
+void DisparityOutputComponent::disparityCb(
+  const stereo_msgs::msg::DisparityImage::SharedPtr disparity_msg)
+{
+  TRACEPOINT(
+    robotperf_image_output_cb_init,
+    "Initializing image output callback",
+    static_cast<const void *>(this),
+    static_cast<const void *>(&(*disparity_msg)));
```

Comment:
@vmayoral, while I was trying to analyze the data I realized my output tracepoint might not be completely correct. I used `robotperf_image_output_cb_init` because I saw that in your A1 benchmark, but where does `robotperf_image_output_cb_init` come from and should it change if I am tracing a disparity message type?